src/debugAdapter/goDebug.ts

```diff
@@ -1160,6 +1178,17 @@ class GoDebugSession extends DebugSession {
 
 	protected evaluateRequest(response: DebugProtocol.EvaluateResponse, args: DebugProtocol.EvaluateArguments): void {
 		verbose('EvaluateRequest');
+		this.evaluateRequestImpl(args).then(out => {
+			const variable = this.delve.isApiV1 ? <DebugVariable>out : (<EvalOut>out).Variable;
+			response.body = this.convertDebugVariableToProtocolVariable(variable);
+			this.sendResponse(response);
+			verbose('EvaluateResponse');
+		}, err => {
+			this.sendErrorResponse(response, 2009, 'Unable to eval expression: "{e}"', { e: err.toString() });
```

Comment:
magic nr ? although i see its used in other places also, there is no constants for them ?